kymatio/scattering2d/tests/test_numpy_scattering2d.py

```diff
@@ -0,0 +1,56 @@
+import os
+import io
+import numpy as np
+from kymatio.scattering2d import Scattering2D_numpy
+import torch
+
+class TestScattering2D_Numpy:
```

Comment:
remove underscore.

Also why is there are class now? Does it require setup and teardown? We had explicitly removed classes from the tests before

Reply:
no, it was mixed! I think it's more consistent(always the same way to test a routine/class) and cleaner(we can use the same test-template) to do a class per "instance" of the backend

Reply:
As far as I can see these classes only cause an indent and have no functionality themselves. So they seem completely redundant

Reply:
I slightly disagree: the indent allows structuring the tests and it is pretty readable. Which format would you recommend? How do we delimit tests addressed to different subparts of the code?

Reply:
I'd recommend adding stuff as

`############## Here we test Scattering2DTorch`

for example.

Unless the classes add something semantically that makes it possible to call e.g. _only the tests from this here class_ using `pytest` command line, I am very against adding such semantic overhead.

Reply:
It is possible: http://doc.pytest.org/en/latest/usage.html

> Another example specifying a test method in the command line:
>
> ```
> pytest test_mod.py::TestClass::test_method
> ```